hpolib/benchmarks/nas/nasbench_101.py

```diff
@@ -169,6 +169,30 @@ def _get_configuration_space(benchmark: Any, seed: Union[int, None] = None) -> C
         cs.seed(seed)
         return cs
 
+    @staticmethod
+    def get_fidelity_space(seed: Union[int, None] = None) -> CS.ConfigurationSpace:
+        """
+        Creates a ConfigSpace.ConfigurationSpace containing all fidelity parameters for
+        the NAS Benchmark 101.
+
+        Parameters
+        ----------
+        seed : int, None
+            Fixing the seed for the ConfigSpace.ConfigurationSpace
+
+        Returns
+        -------
+        ConfigSpace.ConfigurationSpace
+        """
+        seed = seed if seed is not None else np.random.randint(1, 100000)
+        fidel_space = CS.ConfigurationSpace(seed=seed)
+
+        fidel_space.add_hyperparameters([
+            CS.CategoricalHyperparameter('budget', choices=[4, 12, 36, 108], default_value=108)
```

Comment:
This should be a Ordinal

Reply:
Aha, I missed that distinction. Will change it.